tests/test_writer.py

```diff
@@ -389,16 +406,23 @@ def test_write_order(writer, from_line):
     """
     writer.emit_header = False
 
-    packages = [
+    packages = {
         from_line("package_a==0.1"),
         from_line("Package-b==2.3.4"),
         from_line("Package==5.6"),
         from_line("package2==7.8.9"),
-    ]
+    }
     expected_lines = [
         "package==5.6",
         "package-a==0.1",
         "package-b==2.3.4",
         "package2==7.8.9",
     ]
-    assert list(writer._iter_lines(packages)) == expected_lines
+    assert (
+        list(
+            writer._iter_lines(
+                packages, unsafe_requirements=set(), unsafe_packages=set(), markers={}
+            )
+        )
+        == expected_lines
+    )
```

Comment:
```suggestion
    lines = list(
        writer._iter_lines(
            packages, unsafe_requirements=set(), unsafe_packages=set(), markers={}
        )
    )
    assert lines == expected_lines

```


Maybe add an intermediate variable? This looks more readable to me. What do you think?

Reply:
done